docs/variable-tests.md

```diff
@@ -12,5 +13,12 @@ hidden: true
 and
 
 ```js
-const xyz = "<<defvar>>";
+const xyz = '<<defvar>>';
+```
+
+```
+should \<< be escaped >>
```

Comment:
Do you need tests here for `\n` and `\r` cases or are the tests in `@readme/variable` sufficient?

Reply:
Hmm, good call, probably worth it.

Reply:
Technically, `\n` is checked on the next two lines.

Reply:
k. i'm cool to skip it. the tests in the variable package are pretty good as-is.